integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonExtensionSpark2SqlParser.scala

```diff
@@ -27,7 +28,26 @@ import org.apache.spark.sql.catalyst.plans.logical._
 class CarbonExtensionSpark2SqlParser extends CarbonSpark2SqlParser {
 
   override protected lazy val extendedSparkSyntax: Parser[LogicalPlan] =
-    loadDataNew | alterTableAddColumns | explainPlan
+    loadDataNew | alterTableAddColumns | explainPlan | describeColumn | describeShort
```

Comment:
I think, no need to define here, since CarbonExtensionSpark2SqlParser extends CarbonSpark2SqlParser.

Reply:
Done